### PR TITLE
Meta: Ensure new or edited features have Test URLs

### PR DESCRIPTION
**Pull request by fregante**
Most of our features lack test URLs. 

There's no chance of adding them all, but we can try to by reminding ourselves to add them every time we create and change (and _therefore **test**_) a feature.

This will help us progressively add more test URLs

### CODE REVIEW

File: build/verify-test-urls.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		echo ❌ "$FILE"
+		echo "::error file=$FILE,line=1::$ANNOTATION"
+	fi
+done
+
+if [ $EXIT_CODE -eq 1 ]; then
+	echo
+	echo VERIFICATION FAILED, "Test URLs" MISSING
+	exit $EXIT_CODE
+fi
```
Comment (fregante) [inline build/verify-test-urls.sh:43]: Screenshot 3

File: build/verify-test-urls.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+https://github.com/a/REAL/url/here
+https://github.com/another/REAL/url/here
+
+*/
+"
+
+# Replace line breaks with "%0A"
+# https://github.com/actions/toolkit/issues/193#issuecomment-605394935
+ANNOTATION=${ANNOTATION//$'\n'/%0A}
```
Comment (fregante) [inline build/verify-test-urls.sh:24]: Screenshot 4

File: build/verify-test-urls.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+# Automatically exit on error
+set -e
+
+EXIT_CODE=0
+
+# Keep short because GitHub doesn't wrap it
+ANNOTATION="Test URLs missing.
```
Comment (yakov116) [inline build/verify-test-urls.sh:9]: Maybe its just me, the wording sounds flipped
```suggestion
ANNOTATION="Missing Test URLs.
```

Comment (fregante) [reply build/verify-test-urls.sh:9]: No, it’s like VIRUS DETECTED, right? 

“Missing X” would be followed by a list of X, like:

```
Missing keys:

- A
- Q
``` 

but here it works like:

```
# A.tsx

Keys missing (from this file)
```

Comment (yakov116) [reply build/verify-test-urls.sh:9]: Ok

**Pull request merged**